src/test/java/testdata/GetLoginModel.java

```diff
@@ -0,0 +1,19 @@
+package testdata;
+
+import models.LoginModel;
+import staticdata.WebUrl;
+
+public class GetLoginModel {
+    public static LoginModel loginModel() {
+        String password = WebUrl.PASSWORD;
```

Comment:
Here we should have 4 different static methods: 1. returns standard user model 2. returns locked user model 3. problem user 4. performance glitch user